lib/mm_choice.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'mm_outputs'
+
+class Choice
+    class << self
+        def create(query, default_state = true, out = Outputs)
+            @out = out
+            default_display = ""
+            if default_state
+                default_display = "(Y/n)"
+            else
+                default_display = "(y/N)"
+            end
+            question = "#{query} #{default_display}"
+            return pose(question, default_state)
+        end
+
+            private
+
+        def pose(question, default)
+            valid_input = false
+            valid_choice = nil
+            system "clear"
+            until valid_input do
+                @out.ask_question(question)
+                choice = gets.chomp.upcase
+                if choice == "Y" || choice == "N" || choice == ""
```

Comment:
If you're checking each for choice anyway beneath, why not remove the top level and have the else of the elsif blocks be the err clause?